scipy/signal/_fir_filter_design.py

```diff
@@ -1302,79 +1302,41 @@ def fwind1(hsize, window, *, fc=None, fs=2, circular=False):
         Desired window to use for each 1D filter or a single window type 
         for creating circularly symmetric 2-D windows. Each element should be
         a string or tuple of string and parameter values. See
-        '~scipy.signal.get_window' for a list of windows and required
+        `~scipy.signal.get_window` for a list of windows and required
         parameters.
     fc : float or 1-D array_like, optional
-        Cutoff frequency of filter (expressed in the same units as `fs`).
-        Required if `circular` is False.
+        Cutoff frequency of the filter in the same units as `fs`. This defines
+        the frequency at which the filter's gain drops to approximately -6 dB
+        (half power) in a low-pass or high-pass filter. For multi-band filters,
+        `fc` can be an array of cutoff frequencies (i.e., band edges) in the 
+        range [0, fs/2], with each band specified in pairs. Required if 
+        `circular` is False.
     fs : float, optional
         The sampling frequency of the signal. Default is 2.
     circular : bool, optional
         Whether to create a circularly symmetric 2-D window. Default is False.
+    pass_zero : {True, False, 'bandpass', 'lowpass', 'highpass', 'bandstop'}, optional
+        If True, the gain at the frequency 0 (i.e., the "DC gain") is 1.
+        If False, the DC gain is 0. Can also be a string argument for the
+        desired filter type (equivalent to ``btype`` in IIR design functions).
+
+        .. versionadded:: 1.3.0
+           Support for string arguments.
+    scale : bool, optional
+        Set to True to scale the coefficients so that the frequency
+        response is exactly unity at a certain frequency.
+        That frequency is either:
+
+        - 0 (DC) if the first passband starts at 0 (i.e. pass_zero
+          is True)
+        - `fs/2` (the Nyquist frequency) if the first passband ends at
+          `fs/2` (i.e the filter is a single band highpass filter);
+          center of first passband otherwise
```

Comment:
A formulation along the following lines would clarify the 2d nature:
```text
scale : bool, optional
    This parameter is passed to the `firwin` function for each scalar frequency axis.
    Set to ``True`` to scale the coefficients so that the frequency
    response is exactly unity at a certain frequency on one frequency axis.
    That frequency is either:
    
    - 0 (DC) if the first passband starts at 0 (i.e. pass_zero is ``True``)
    - `fs/2` (the Nyquist frequency) if the first passband ends at
       `fs/2` (i.e the filter is a single band highpass filter);
       center of first passband otherwise

```